linked_list/linked_list.py

```diff
@@ -23,13 +23,53 @@ def __init__(self):
     self.tail = None
 
   def add_to_tail(self, value):
-    pass
+    node = Node(value)
+    if self.head == None:
+      self.head = node
+      self.tail = node
+    else:
+      self.tail.set_next(node)
+      self.tail = node
 
   def remove_head(self):
-    pass
+    if self.head == None:
+      return self.head
+    else:
+      temp = self.head.get_value()
+      # self.head = temp
+      self.head = self.head.get_next()
+      if self.head == None:
+        self.tail = None
+      return temp
 
   def contains(self, value):
-    pass
+    current = self.head
+    if self.head is None:
+      return False
+    else:
+      while current != None:
```

Comment:
I would pick either `=` or `is` when checking for `None` and use it throughout. `is` is the preferred way https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_none.html